api-stat.go

```diff
@@ -49,6 +49,31 @@ func (c Client) BucketExists(bucketName string) (bool, error) {
 	return true, nil
 }
 
+// List of header keys to be filtered, usually
+// from all S3 API http responses.
+var defaultFilterKeys = []string{
+	"Transfer-Encoding",
+	"Accept-Ranges",
+	"Date",
+	"Server",
+	"Vary",
+	"x-amz-request-id",
```

Comment:
let me use the helper method .Del() instead.  which will canonicalize this properly.